config/locales/en/errors.yml

```diff
@@ -432,3 +417,13 @@ en:
               invalid: |
                 Name must only include letters a to z, and special characters such as hyphens, spaces and apostrophes.
                 Name must be 2 words or more.
+      messages:
```

Comment:
how does this work to link the fields?

Reply:
If ActiveModel can't find a message for a specific error type on a specific field it turns out it falls back to searching `language.active_model.errors.messages.error_type`. Obviously for error types like `blank`, `greater_than` etc there's no point having a fallback because the message will be so different for different fields. But as we're using some very specific, explicit error type names, like `blank_hours`, we can provide sensible defaults here and only override them if needed.

I was going to use the `&shared_time_errors` / `<<: *shared_time_errors` pattern, but that only works if the shared fields are all in the same file, and we recently took the decision to start splitting errors out of one massive `errors.yml` file, so the time errors in `alternative_quotes.yml` couldn't be shared that way.